src/sentry/utils/snuba.py

```diff
@@ -147,6 +147,9 @@ def log_snuba_info(content):
     "http.decoded_response_content_length": "sentry_tags[http.decoded_response_content_length]",
     "http.response_transfer_size": "sentry_tags[http.response_transfer_size]",
     "app_start_type": "sentry_tags[app_start_type]",
+    "replay.id": "sentry_tags[replay_id]",
+    "browser.name": "sentry_tags[browser.name]",
+    "origin.transaction": "sentry_tags[transaction]",
```

Comment:
`transaction` is already being used for the `segment_name` field.

Reply:
Actually, QQ. Why can't we just go ahead and map `origin.transaction` to `segment_name`? Would it be faster to access the segment name column instead of the sentry tags column for this?

Reply:
I need to get `sentry_tags[transaction]` which is a different tag value from the `segment_name` field.

INP spans don't have a `segment_name` because we don't connect those spans to a parent pageload or navigation  (because it doesn't technically belong to those spans).

But we do attach a `sentry_tags[transaction]` tag to INP spans so we can tell which page an interaction occurred in, without having to create a parent-child span relationship.